src/index.css

```diff
@@ -18,6 +18,9 @@
   background-color: #4ad3b3;
   color: white;
 }
+.my-div{
```

Comment:
i told you how to write css yesterday, for each component/ container.
can you follow that and also ask other to follow same.
dont use random classes